I found a few possible errors, one which is breaking for me

So I ignored a few of the issues since, this project is still early, and they were only type issues that I could work around, but with version 0.3.0 another issue has cropped up that breaks things for me.

So the major issue that breaks things for me now, is the fact then after updating to 0.3.0 I am presented with
```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './errors' is not defined by "exports" in /workspace/node_modules/@nahkies/typescript-koa-runtime/package.json
```

and the small but workaround issues that I have found since day one of using this, is also related in a way to the above, but not caused by it. 
```
Cannot find module '@nahkies/typescript-koa-runtime/server' or its corresponding type declarations.ts(2307)
Cannot find module '@nahkies/typescript-koa-runtime/zod' or its corresponding type declarations.ts(2307)
```
it seems the runtime has no issues with these paths, but typescript does, and adding `/src/` after `@nahkies/typescript-koa-runtime` path fixes the typescript issues, so it's just a simple path rewrite that needs to happen for the typescript types.

One last possible issue that I think you overlooked was you are linking `joi` definitions for `zod`.
https://github.com/mnahkies/openapi-code-generator/blob/6d6183abf264f085eec4cb416bd8e50db11432e6/packages/typescript-koa-runtime/package.json#L37

It's possible one or more of these issues are related to my setup, so do let me know, thanks.

--- a/packages/typescript-koa-runtime/package.json
+++ b/packages/typescript-koa-runtime/package.json
@@ -34,7 +34,7 @@
     "./zod": {
       "require": "./dist/zod.js",
       "import": "./dist/zod.js",
-      "types": "./dist/joi.d.ts"
+      "types": "./dist/zod.d.ts"
     }
   },
   "scripts": {

